test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -345,6 +345,9 @@ function compute-kube-apiserver-params {
 	else
 		params+=" --etcd-servers=${ETCD_SERVERS}"
 	fi
+	if [[ "${ETCD_QUORUM_READ:-}" ]]; then
```

Comment:
Alternatively you can use `APISERVER_TEST_ARGS` (see above) to pass this. I'm fine with both ways.